Mathlib/LinearAlgebra/QuadraticForm/Basic.lean

```diff
@@ -1105,34 +1138,36 @@ The determinant of the matrix is the discriminant of the quadratic form.
 variable {n : Type w} [Fintype n] [DecidableEq n]
 variable [CommRing R] [AddCommMonoid M] [Module R M]
 
-/-- `M.toQuadraticForm'` is the map `fun x ↦ col x * M * row x` as a quadratic form. -/
-def Matrix.toQuadraticForm' (M : Matrix n n R) : QuadraticForm R (n → R) :=
-  LinearMap.BilinForm.toQuadraticForm (Matrix.toLinearMap₂' M)
-#align matrix.to_quadratic_form' Matrix.toQuadraticForm'
+variable (M : Matrix n n R)
```

Comment:
The variable name you use here is unfortunate. Change its name.

Every theorem and definition in this section still applies to a matrix whose entries lie in an `R`-module.